elastic/kibana
# Fixes render glitch on space selector screen

## Summary

When scrollbars are always on, the render of the background image on the space selector screen was part
<img width="663" alt="Screenshot 2023-02-15 at 6 19 09 PM" src="https://user-images.githubusercontent.com/103939324/219214315-529a38d8-3cf1-4ce7-8f1e-ca84ffc731bf.png">
ially overlapping the scroll bar in Chrome and Firefox.


This was fixed by overriding the position setting of the kibanaFullScreenGraphics mixin in .spcSpaceSelector.
<img width="669" alt="Screenshot 2023-02-15 at 6 17 52 PM" src="https://user-images.githubusercontent.com/103939324/219214099-b317fb53-28aa-4e60-b648-bb98757330c5.png">

Additionally, it was noticed that the telemetry banner, if not dismissed, had been visually hidden in this screen when it should have been shown. This is now also resolved.

### Test
- Set scrollbars to always on (MacOS: Settings->Appearance->Show scroll bars->Always)
- Start ES & Kibana - do NOT dismiss the telemetry banner
- Add multiple spaces
- Navigate to [base URL]/spaces/space_selector
- Adjust window size to cause scrollbar to appear and background graphics to align with it (reference above screenshot)
- The background graphics should not render over the scroll bar
- The telemetry banner should render in front and at the top of the window
